internal/functions/new/templates/index.ts

```diff
@@ -16,8 +16,15 @@ Deno.serve(async (req) => {
   )
 })
 
-// To invoke locally:
-// curl -i --location --request POST 'http://127.0.0.1:{{ .Port }}/functions/v1/{{ .Slug }}' \
-//   --header 'Authorization: Bearer {{ .Token }}' \
-//   --header 'Content-Type: application/json' \
-//   --data '{"name":"Functions"}'
+/* To invoke locally:
+
+  1. Run `supabase start`
+  2. Run `supabase functions serve`
```

Comment:
don't need `functions serve` if you do `start`

Reply:
@laktek you're right, updated pr.